### ISSUE TITLE
Crash on IE11

### ISSUE THREAD

**Issue by saostad** [mironov/react-redux-loading-bar#65]
Hi, 

My app works on all browsers except IE11, which for some reasons I have to support it. Here is the error message from console:

```
DOM7011: The code on this page disabled back and forward caching. For more information, see: http://go.microsoft.com/fwlink/?LinkID=291337
localhost:3000
HTML1300: Navigation occurred.
localhost:3000
The above error occurred in the <LoadingBar> component:
    in LoadingBar (created by Connect(LoadingBar))
    in Connect(LoadingBar) (at main.component.js:20)
    in Router (created by BrowserRouter)
    in BrowserRouter (at main.component.js:18)
    in Provider (at main.component.js:17)
    in Main (at App.js:40)
    in ApolloProvider (at App.js:39)
    in App (at index.js:13)

React will try to recreate this component tree from scratch using the error boundary you provided, App.
SCRIPT438: Object doesn't support property or method 'includes'
bundle.js (85239,9)

```

It seams problem is related to includes method for objects which is not available in IE11, I find a stack overflow [question ](https://stackoverflow.com/questions/31119300/ie11-object-doesnt-support-property-or-method-includes-javascript-window?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa) about it but I not expert on that area. 
can you please check it maybe it can help to fix the problem? 

BTW thanks for awesome work 👍

**Comment by mironov** [comment]
@saostad Thank you for reporting this.

I published v4.0.5 that should fix the problem for you. Please note that I haven't tested it in IE11 and there might be other issues.

**Comment by saostad** [comment]
thanks, it seems there no problem anymore.
